### PR TITLE
Sphinx based documentation

### PR DESCRIPTION
**Pull request by valberg**
As per https://github.com/jazzband/django-invitations/issues/176 (and https://github.com/jazzband/help/issues/237#issuecomment-1023124661) we should have some better docs for django-invitations. This is the first step in that direction.

### CODE REVIEW

**Review by brylie — APPROVED** [review]

File: docs/index.rst [hunk 1]
```diff
@@ -0,0 +1,23 @@
+django-htmx documentation
```
Comment (MrCordeiro) [inline docs/index.rst:1]: django-htmx?

File: docs/changelog.rst [hunk 2]
```diff
@@ -0,0 +1,23 @@
+Changelog
+=========
+
```
Comment (MrCordeiro) [inline docs/changelog.rst:3]: I would remove this Changelog as it's not up to date with the CHANGELOG file

File: README.md [hunk 3]
```diff
@@ -8,187 +8,12 @@
...
-
-
-### Bulk Invites
-
-Bulk invites are supported via JSON.  Post a list of comma separated emails to the dedicated URL and Invitations will return a data object containing a list of valid and invalid invitations.
-
-
-### Testing
-
-`python manage.py test` or `tox`
```
Comment (MrCordeiro) [inline README.md:117]: Testing needs to be ported to the Usage docs

**Review by MrCordeiro — COMMENTED** [review]
Overall very good work. Made a few comments..

File: docs/index.rst (hunk 1 above)
Comment (valberg) [reply docs/index.rst:1]: Embarrassing! I copied the documentation from django-htmx as a starting point.

File: docs/changelog.rst (hunk 2 above)
Comment (valberg) [reply docs/changelog.rst:3]: Ah well, I copied the `ChangeLog` file and removed the, for me, hard to parse parts, or if the parts were only simple lines like "Travis.". In the end I decided only to include the latest changelog. 

If you think we should just add the whole file I'm fine with that.

File: README.md (hunk 3 above)
Comment (valberg) [reply README.md:117]: Would a "contribute.rst" make sense? This does not make sense, to me, to put in "Usage" since it is not about usage, but about development of django-invitations.

Comment (MrCordeiro) [reply README.md:117]: For sure. It's also a section that needs more work. For now, at least we should confirm that the test command works and that people should run tox before submitting a PR.

This is how I put it on my fork:

---

### Testing

It's important that any new code is tested before submission. To quickly test code in your active development environment, you should first install all of the requirements by running:

```bash
source .venv/bin/activate
pip install -e '.[testing]' -U  # I had replaced setup.py and MANIFEST for a toml file
```

Then, run the following command to execute tests:

```bash
pytest --cov-report term --cov=invitations  --ignore=tests/allauth/  --ds=tests.settings tests
```

To test the integration with django-allauth, first make sure you have it installed. Then run:

```bash
pytest --cov-report term --cov=invitations  --ignore=tests/basic/  --ds=tests.settings_allauth tests
```

Testing in a single environment is a quick and easy way to identify obvious issues with your code. However, it's important to test changes in other environments too, before they are submitted. In order to help with this, django-invitations is configured to use tox for multi-environment tests. They take longer to complete, but can be triggered with a simple command:

```bash
tox
```

File: docs/changelog.rst (hunk 2 above)
Comment (MrCordeiro) [reply docs/changelog.rst:3]: Works for me

File: docs/index.rst (hunk 1 above)
Comment (MrCordeiro) [reply docs/index.rst:1]: Only this one's left now! Keep on keeping on!

**Review by MrCordeiro — COMMENTED** [review]
Almost theeeeere! Just need to use the correct package name on index.rst

**Review by MrCordeiro — APPROVED** [review]
